ouroboros-consensus/test-consensus/Test/Dynamic/RealPBFT.hs

```diff
@@ -177,4 +198,115 @@ generateGenesisConfig (NumCoreNodes n) =
               -- The nodes are the richmen
               { Genesis.tboRichmen = fromIntegral n }
         }
+      , Genesis.gsK = Common.BlockCount $ maxRollbacks pbftSecurityParam
```

Comment:
This was 10 before, but now it's 1) easier to see that and 2) easy to vary it if we want.